string status
float64 theta1
float64 theta2
float64 theta3
